susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1497,6 +1513,22 @@
         'BASECHANNEL' : 'almalinux8-aarch64', 'PKGLIST' : RES8,
         'DEST' : DOCUMENT_ROOT + '/pub/repositories/almalinux/8/bootstrap/'
     },
+    'almalinux-9-x96_64-uyuni' : {
```

Comment:
Looks like there is a typo in the AL9 architecture part, `x96_64` should be `x86_64`. Not sure if this really affects anything, but I guess it's worth a mention.

https://github.com/uyuni-project/uyuni/blob/b69b5582a4dffc83e69cf53e6058a1bb4bf19d08/susemanager/src/mgr_bootstrap_data.py#L1516